### PR TITLE
Update release notes for 2016-05-05

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-2285e8b098/). :bow:

File: _reference/2015-10-17-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ topics:
   - docker
 ---
 
+### May 5, 2016
+
+#### What's new
+
+* Improved error messages for [common errors](https://getcarina.com/docs/reference/error-codes/) related to security restrictions
```
Comment (everett-toews) [inline _reference/2015-10-17-release-notes.md]: Please use the title of the page as the text for the link.

**Comment by everett-toews** [comment]
This is awesome. I'm so looking forward to this.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-66bd1633a5/).

**Comment by lblackstone** [comment]
@everett-toews Fixed the link

**Pull request merged**